Grpc.AspNetCore.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.136
+# Visual Studio Version 16
```

Comment:
is switch to VS2019 necessary for this PR?

Reply:
No but our infrastructure is moving towards requiring VS2019 soon so I updated the version of VS on my machine. I think @JamesNK has already made the switch and I noticed our PRs were ping ponging between the two. There isn't much harm in this though as I was able to open these solutions in VS2017 as well.